index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio</title>
+  <script src="https://kit.fontawesome.com/9dfa895fe8.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+  <div class="content">
+    <div class="navbar">
+      <div class="logo">
+        <h2>My Logo</h2>
+      </div>
+      <div class="icon">
+        <img src="images/Union.jpg" alt="Union">
+      </div>
+    </div>
+    <div class="section">
+      <img class="img1" src="images/Ellipse11.jpg" alt="Ellipse11">
+      <img class="img2" src="images/Ellipse12.svg" alt="Ellipse12">
+      <img class="img3" src="images/Ellipse8.svg" alt="Ellipse8">
+      <img class="img4" src="images/Ellipse4.svg" alt="Ellipse4">
+      <img class="img5" src="images/Ellipse5.svg" alt="Ellipse5">
+      <img class="img6" src="images/Rectangle7.svg" alt="Rectangle7">
+      <img class="img7" src="images/Ellipse3.jpg" alt="Ellipse 3">
+      
+      <div class="title">
+        <h1>I’m Ilham
+          Glad to see you!</h1>
+      </div>
+      <div class="para">
+        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my
+          work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact me.</p>
+      </div>
+      <h3 class="h3">LET’S CONNECT</h3>
+      <div class="icons">
+        <ul class="headline-img">
+          <li>
+            <a href="#"><img src="images/Vector(3).svg" alt="twitter-icon" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="images/Linkedinicon.svg" alt="linkdn-icon" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="images/Vector(2).svg" alt="monster-icon" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="images/Vector(1).svg" alt="gitbubicon" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="images/Vector.svg" alt="thumb" /></a>
+          </li>
+        </ul>
+      </div>
```

Comment:
- [ ] The social images are broken, I would recommend fixing the names of images in the image folder by removing the white space. (ex: `Linkedin icons.svg` should be `Linkedinicons.svg` or `Linkedin-icons.svg`):

<img width="203" alt="image" src="https://user-images.githubusercontent.com/77190203/159253692-aef467ca-27f0-4ae9-997a-be13e2966136.png">